File: frontend/src/utils/specialties 2.ts
// Mirrors backend/app/services/specialties.py — six curated clinical
// specialties, each backed by an explicit MeSH-scoped query server-side.
export interface Specialty {
  key: string
  label: string
}

export const SPECIALTIES: Specialty[] = [
  { key: 'cardiology', label: 'Cardiology' },
  { key: 'oncology', label: 'Oncology / Cancer' },
  { key: 'infectious_disease', label: 'Infectious Disease' },
  { key: 'neurology', label: 'Neurology' },
  { key: 'alzheimers_dementia', label: "Alzheimer's & Dementia" },
  { key: 'public_health_policy', label: 'Public Health & Policy' },
]

export const DEFAULT_SPECIALTY = SPECIALTIES[0].key

File: backend/app/services/specialties 2.py
# Each specialty maps to an explicit MeSH-scoped PubMed query string.
SPECIALTY_QUERIES: dict[str, str] = {
    "cardiology": '"Cardiovascular Diseases"[Mesh]',
    "oncology": '"Neoplasms"[Mesh]',
    "infectious_disease": '"Communicable Diseases"[Mesh]',
    "neurology": '"Nervous System Diseases"[Mesh]',
    "alzheimers_dementia": '"Alzheimer Disease"[Mesh] OR "Dementia"[Mesh]',
    "public_health_policy": '"Public Health"[Mesh] OR "Health Policy"[Mesh]',
}

SPECIALTY_LABELS: dict[str, str] = {
    "cardiology": "Cardiology",
    "oncology": "Oncology / Cancer",
    "infectious_disease": "Infectious Disease",
    "neurology": "Neurology",
    "alzheimers_dementia": "Alzheimer's & Dementia",
    "public_health_policy": "Public Health & Policy",
}
